### PR TITLE
Bower.rst update

### LINKED ISSUE #980: update "Manage Client-Side Packages with Bower"
https://docs.asp.net/en/latest/client-side/bower.html
- [x] update
- [x] answer questions (You should explain the difference between "4.6.4", "^4.6.4" and "~4.6.4")

### LINKED ISSUE #1244: bower.json and package.json not visible in VS solution explorer
See https://docs.asp.net/en/latest/client-side/bower.html#comment-2682384620
in https://docs.asp.net/en/latest/migration/mvc.html we have them edit both files.
Need to document how to work with these files and why they are hidden.

### PR DESCRIPTION
**Pull request by spboyer**
Fixes #980 
Fixes #1244

### CODE REVIEW

**Comment by spboyer** [comment]
@danroth27

File: aspnet/client-side/bower.rst [hunk 1]
```diff
@@ -59,43 +59,65 @@ Now that the installation step has been completed, expand the twisty to the left
...
+    </div>
+    
+Run the application to see the changes.
+
+  .. image:: bower/_static/app-with-icons.png
 
 Exploring the Client Build Process
 ----------------------------------
-
+
```
Comment (danroth27) [inline aspnet/client-side/bower.rst]: Remove unnecessary whitespace.

File: aspnet/client-side/bower/samples/Starter/.bowerrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (danroth27) [inline aspnet/client-side/bower/samples/Starter/.bowerrc]: I don't think this sample is needed at all. I don't see it referenced anywhere.

Comment (spboyer) [inline aspnet/client-side/bower/samples/Starter/.bowerrc]: the `bowerc` file? or the entire Starter sample? The Starter sample is what is used in the entire beginning of the article of how to add the resources etc. 

There is not a link to the sample code if that is what you mean

Comment (danroth27) [inline aspnet/client-side/bower/samples/Starter/.bowerrc]: I mean the entire sample. If you don't have any literal-includes from the sample code then I don't see any reason to have it in the repo.

Comment (spboyer) [inline aspnet/client-side/bower/samples/Starter/.bowerrc]: ok removed

**Comment by spboyer** [comment]
ok, after a minor oops. should be good @danroth27

File: aspnet/client-side/bower.rst [hunk 3]
```diff
@@ -59,43 +58,64 @@ Now that the installation step has been completed, expand the twisty to the left
...
+        </ul>
+    </div>
+    
+Run the application to see the changes.
+
+  .. image:: bower/_static/app-with-icons.png
 
 Exploring the Client Build Process
 ----------------------------------
-
```
Comment (danroth27) [inline aspnet/client-side/bower.rst]: Nit: There should still be a newline here. Just no spaces.

File: aspnet/client-side/bower.rst [hunk 4]
```diff
@@ -3,51 +3,50 @@
...
+ASP.NET Core project templates pre-construct the client build process for you. The ubiquitous jQuery and Bootstrap packages are installed, and the plumbing for Bower is already in place. The screenshot below depicts the initial project in Solution Explorer. It's important to enable the "Show All Files" option, as the bower.json file is hidden by default.
 
 .. image:: bower/_static/mvc-project.png
   :width: 300px
 
 Client-side packages are listed in the bower.json file. The ASP.NET Core project template pre-configures bower.json with jQuery, jQuery validation, and Bootstrap. 
 
-Let’s add support for photo albums by installing the `Fotorama <http://fotorama.io/>`_ jQuery plugin. Bower packages can be installed either via the Manage Bower Packages UI or manually in the bower.json file.
+Let’s add support for `Font Awesome <http://fontawesome.io>`_ to add some scalable vector css icons to the HTML. Bower packages can be installed either via the Manage Bower Packages UI or manually in the bower.json file.
```
Comment (danroth27) [inline aspnet/client-side/bower.rst]: Just curious - why the change to Font Awesome?

File: aspnet/client-side/bower.rst [hunk 5]
```diff
@@ -59,43 +58,64 @@ Now that the installation step has been completed, expand the twisty to the left
...
+        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
+              asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
+              asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
+        <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+              asp-fallback-href="~/lib/font-awesome/css/font-awesome.min.css" />
+      </environment>
 
-.. image:: bower/_static/photo-gallery.png
+Open ``Views\Home\Index.cshtml`` and add the icons to the **How to**, **Overview**, and **Run & Deploy** headers. You'' notice when typing, intellisense is available on the css classes.
```
Comment (danroth27) [inline aspnet/client-side/bower.rst]: `You''` ?

Comment (danroth27) [inline aspnet/client-side/bower.rst]: @Rick-Anderson intellisense -> IntelliSense?

File: aspnet/client-side/bower.rst (hunk 4 above)
Comment (spboyer) [inline aspnet/client-side/bower.rst]: The Manage Bower Packages viewer only shows the top 1000 packages. Fotorama no longer meets that list. Font-Awesome is a top 100.

File: aspnet/client-side/bower.rst (hunk 5 above)
Comment (spboyer) [inline aspnet/client-side/bower.rst]: yes it should be intelliSense
